script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -307,12 +325,12 @@ In an instant, I forget all about the pain caused by my elevated heart rate.
 [sha:8b1619a381c7ddcb087f6b3dbde0be774831de0d]{
 -- Page 397, Offset 9625.
 -- ―――燃えている。
-―――It's burning.
+It's on fire...
```

Comment:
probably could be a bit more dramatic
>The fire has consumed everything.

Reply:
Hmm that's kinda just stealing the next line's thunder